sormas-app/app/src/main/java/de/symeda/sormas/app/util/DataUtils.java

```diff
@@ -75,6 +75,11 @@ public static List<Item> getBooleanItems() {
 		return list;
 	}
 
+	public static boolean emptyOrWithOneNullItem(List<Item> listIn) {
+		return listIn.isEmpty() || (listIn.size() == 1 && listIn.get(0).getValue() == null);
+	}
```

Comment:
i think here would be better to check that the list is empty or all elements in list are null